README.md

```diff
@@ -3,11 +3,38 @@ A [Kanister](https://github.com/kanisterio/kanister) backup management system.
 
 Taweret sets retention periods for Kanister backups and deletes them once they expire by interacting with Kanister CRDs.
```

Comment:
I find the above a little confusing; can it make sense to have a specified retention period without deletion?

Reply:
I don't really understand what you mean, sorry. If a backup is older than the specified retention period, it is deleted.

Reply:
I find the wording confusing - perhaps I'm either being stupid or nit-picking ;-)

`Tawaret sets retention periods for Kanister backups` - I (probably wrongly) interpreted this as meaning that Tawaret sets something inside Kanister and my q then was what happens if this is set but there is no Tawaret to delete it.

Maybe changing it to `Tawaret defines retention periods for Kanister backups` would indicate more that these retention periods are owned by Tawaret and not Kanister...